jxl/src/color/tf.rs

```diff
@@ -138,6 +164,19 @@ mod test {
         });
     }
 
+    #[test]
+    fn bt709_roundtrip_arb() {
+        arbtest::arbtest(|u| {
+            let samples: Vec<f32> = arb_samples(u)?;
+            let mut output = samples.clone();
+
+            linear_to_bt709(&mut output);
+            bt709_to_linear(&mut output);
+            assert_all_almost_eq!(&output, &samples, 5e-6);
```

Comment:
I'm not sure about the max error here...

Reply:
A good way to find out is to run tests in release mode with ARBTEST_BUDGET_MS=100000 and see if it fails :-)